README.md

```diff
@@ -1,2 +1,18 @@
 # BonnyCI Documentation
 [![Build Status](https://travis-ci.org/BonnyCI/docs.svg?branch=master)](https://travis-ci.org/BonnyCI/docs)
+
+## Table of Contents
+- [Introduction](#introduction)
+- [Getting Started](#getting-started)
+- [Contributing](#contributing)
+
+## Introduction
+BonnyCI is a Continuous Integration service offered to developers of Open Source projects.
+
+## Getting Started
+Checkout the [getting started](getting_started) section.
+
+## Contributing
```

Comment:
I agree whole-heartedly. I was trying to get the basics in place for us to be able to move forward with the rest of the documentation work.